### PR TITLE
Performance audit

### PR DESCRIPTION
**Pull request by brahimdidi**
In this part, I added huge data into the owners and visits table to test the performance, then created Indexes on columns to speed up the performance.

## Requierments
Run the query to add an extra column to the owners table:
Run the statements to add data to your database (executing them might take a few minutes).
Use EXPLAIN ANALYZE on the previous queries to check what is happening.
Find a way to decrease the execution time of the first query.
Find a way to improve the execution time of the other two queries.
## Screenshots

Screenshot (48)
Screenshot (50)

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -53,4 +53,16 @@ CREATE TABLE specializations (
...
\ No newline at end of file
+ );
+
+ -- PERFORMENCE TEST PART
+
+-- ADD NEW email COULUMN INSIDE TABLE owners.
+
+ ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+-- CREATE INDEXES ON COLUMNS TO SPEED UP THE PERFORMENCE.
```
Comment (Mre55) [inline schema.sql:64]: Make sure add proper database performance audit implementation for `SELECT * FROM visits where vet_id = 2;` query. I see the execution time after implementing database performance audit still greater than 1000ms.

**Review by Mre55 — CHANGES_REQUESTED** [review]
**Hi @brahimdidi,**

You have done a great job so far! 💪 👏 
You still need to work on a few issues before moving on to the next project, but you're almost there!

## To highlight 🎯

- The Pull Request is following the Git Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 

## Required Changes ♻️🛠️ 
_Check the comments under the review._




## Optional suggestions 🟡 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (brahimdidi) [reply schema.sql:64]: Do you have any suggetions, I crated index but it still uses seq scan.

**Comment by brahimdidi** [comment]
Screenshot (53)

**Review by OLIPLICHE — COMMENTED** [review]
Hi @brahimdidi , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] Work and submission mode: In reading your Project requirements, I have pasted below what was highlighted.

- You should implement the above requirements only in one repository in your **pair-programming group.**
- You should ask for a review and **submit this activity on behalf of your pair-programming group**.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by brahimdidi** [comment]
THIS WAS IMPLEMENTED ONLY IN ONE REPOSITORY , MY PARTNER IS NOT SHOWING UP

**Comment by brahimdidi** [comment]
As I said my partner is leaving microverse she's been sick the whole week and I have to work alone.

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi team, 
As I can clearly see that you had some fun with the explain analyze and I am sure that in the future you will know how to start the investigation for some slow DB queries, it's time to approve this project 

There is nothing else to say other than... it's time to merge it!
Congratulations!

## Happy Coding 👏👏👏
#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**